Example/Example/ViewController.m

```diff
@@ -43,6 +45,11 @@ - (void)viewDidLoad {
                           IDENTIFIER_TYPE_KLAVIYO_ID: @"555555",
                           IDENTIFIER_TYPE_CUSTOM_IDENTIFIERS: @{@"customId": @"customIdValue"}
     };
+    [sdk identify:_userIdentifiers];
+    
+    
+    // Random Example app stuff
```

Comment:
nit: should we clarify this comment a little?

Reply:
you're right, this wasn't clear. Updated to a hopefully more clear comment